spec/models/user_spec.rb

```diff
@@ -8,6 +8,13 @@
   it { should validate_presence_of(:email) }
   it { should validate_presence_of(:password).on(:create) }
 
+  it { should allow_value("(555) 555-5555").for(:phone_number) }
+  it { should allow_value("555-555-5555").for(:phone_number) }
+  it { should allow_value(nil).for(:phone_number) }
+  it { should allow_value("").for(:phone_number) }
```

Comment:
@jferris providing a phone number is entirely optional. 

It doesn't seem to require an `allow_empty` or `allow_nil` option.